From a096f9cf699e839a545b56e14f0687b928de4a2d Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Sat, 25 Jan 2025 19:50:05 +0100
Subject: [PATCH] Drop build date from generated file

for reproducible builds.

Without this, dagcheck.c in the debugsource package
varies across builds.
---
 code/tools/lcc/lburg/lburg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/code/tools/lcc/lburg/lburg.c b/code/tools/lcc/lburg/lburg.c
index 7cf859ff77..c3bdb1bfda 100644
--- a/code/tools/lcc/lburg/lburg.c
+++ b/code/tools/lcc/lburg/lburg.c
@@ -458,7 +458,7 @@ static void emitdefs(Nonterm nts, int ntnumber) {
 static void emitheader(void) {
 	time_t timer = time(NULL);
 
-	print("/*\ngenerated at %sby %s\n*/\n", ctime(&timer), rcsid);
+	print("/*\ngenerated by %s\n*/\n", rcsid);
 	print("static void %Pkids(NODEPTR_TYPE, int, NODEPTR_TYPE[]);\n");
 	print("static void %Plabel(NODEPTR_TYPE);\n");
 	print("static int %Prule(void*, int);\n\n");
